scripts/populate_mock_db.py

```
@@ -0,0 +1,60 @@
+        role = ["WC", "CM", "CC", "FFC", "FFD", "FFT"][i % 6]
+        skills = "BA TTR LGV" if i % 4 == 0 else "BA TTR" if i % 2 == 0 else "BA"
+        contract_hours = "42.00 hrs"
+        crew_data.append((name, role, skills, contract_hours))
+
+    c.executemany(
+        "INSERT INTO crew (name, role, skills, contract_hours) VALUES (?, ?, ?, ?)",
+        crew_data,
+    )
+
+    # Add availability
+    now = datetime.now()
```

Ruff (0.15.1)

[warning] `datetime.datetime.now()` called without a `tz` argument

(DTZ005)